src/app.js

```diff
@@ -39,7 +67,8 @@ app.get('/', (req, res) => {
     current_time: new Date()
   });
 });
-app.get('/v1/agencies/:reportAgency/reports/:reportName/data', fetchData);
-app.get('/v1/reports/:reportName/data', fetchData);
+app.get('/v1.1/domain/:domain/reports/:reportName/data', checkDomainFilter);
```

Comment:
Do we feel like we have to bump the API version to add a query param? That may be a bit more disruptive then not doing it since with this change we are no longer supporting v1.